documentation/docs/release-notes/3.1.0.md

```diff
@@ -0,0 +1,170 @@
+
+![New Mongo query fingerprint](../images/Mongo_fingerprint.png)
+
+#### Enhanced query examples visualization
+We've refined the query examples interface in Mongo QAN by removing unnecessary enumeration labels ("1 item", "2 item", etc.) and upgrading the JSON viewer component. 
+
+This creates a cleaner display that emphasizes the actual query content rather than distracting metadata.
+
+#### Improved MongoDB cluster topology visualization
+
+We have enhanced the [MongoDB Cluster Summary dashboard](../reference/dashboards/dashboard-mongodb-cluster-summary.md) with a redesigned **Current Topology** section. 
+
+This section now properly categorizes MongoDB cluster components—shards, config servers, and mongos routers—within their appropriate hierarchical sections. This replaces the previous implementation that incorrectly grouped all components under the **Config Servers** section.
+
+![Topology section](../images/Topology.png)
+
+#### Enhanced MongoDB dashboard visualizations
+Based on community feedback, we've made several improvements to the [MongoDB ReplicaSet Summary](../reference/dashboards/dashboard-mongodb-replset-summary.md) and [MongoDB Sharded Cluster Summary](../reference/dashboards/dashboard-mongodb-cluster-summary.md) dashboards:
+
+- improved detection of node states to better identify down nodes
+- optimized **Top Hottest Collections** panel showing only the most critical data
+- improved readability with rounded numbers in **Overview** panels
+- increased precision with decimal values (tenths) in **Top Hottest Collections** panels
+- resolved visualization issues in **Disk I/O** and **Network traffic** panels for more reliable performance monitoring
+
+#### Enhanced filtering capabilities 
+We've improved the filtering experience on the [MongoDB Instances Compare](../reference/dashboards/dashboard-mongodb-instances-compare.md) dashboard with four new selection boxes in addition to the existing **Service Name** filter. These filters allow you to narrow down instances by:
+
+- **Replication Set**
+- **Environment**
+- **Cluster**
+- **Node** 
+
+![MongoDB Instances Compare](../images/New_Filters.png)
+
+These additional filters make it easier to navigate and compare MongoDB instances in larger environments, enabling you to quickly narrow down the specific instances you want to monitor.
+
+### Enhanced role-based access control
+
+We've refined role-based access for Advisors and Alerts, making it easier for teams to collaborate while maintaining security boundaries:
+
+#### Advisors access for Editors
+
+Users with the Editor role can now access Advisors insights without requiring Admin privileges. This is especially beneficial in managed service environments where Admin access is typically restricted to database administrators.
+
+While Editors can view Advisors insights, only Admins can run, disable, or edit Advisors and execute checks.
```

Comment:
What is the difference between run and execute checks?

Reply:
No difference. let's remove "run"